bin/bashcov

```diff
@@ -13,8 +13,33 @@ coverage = runner.result
 
 require "simplecov"
 
-SimpleCov.command_name Bashcov.name
+SimpleCov.coverage_dir(ENV["COVERAGE_DIR"]) if ENV["COVERAGE_DIR"]
+
+if ENV["COVERAGE_NAME"]
+  SimpleCov.command_name ENV["COVERAGE_NAME"]
+else
+  SimpleCov.command_name Bashcov.name
+end
+
+reports = Bashcov.options.reports
+unless reports.empty?
+  formatters = [
+    SimpleCov::Formatter::HTMLFormatter
+  ]
+  reports.each do |r|
+    require r[:require]
+    formatters.push(r[:formatter].split("::").inject(Object) { |a, e| a.const_get e })
+  end
+  SimpleCov.formatters = formatters
+end
```

Comment:
I think https://github.com/infertux/bashcov/pull/13#discussion-diff-42921360R26 redefine all formatters. That's why for example doesn't work. If it's possible I would like to have both approach. git-ignore needs .simplecov because of use of test suite sharness, but in general people should use bashcov as another bash script. Anyway, I can keep this on my own fork till it's mature. I understand your are reluctant to add this